test/e2e/storage/persistent_volumes-local.go

```diff
@@ -1344,6 +1345,7 @@ func createSecPod(config *localTestConfig, volume *localTestVolume, hostIPC bool
 	pod, err := framework.CreateSecPod(config.client, config.ns, []*v1.PersistentVolumeClaim{volume.pvc}, false, "", hostIPC, hostPID, seLinuxLabel, nil, framework.PodStartShortTimeout)
 	podNodeName, podNodeNameErr := podNodeName(config, pod)
 	Expect(podNodeNameErr).NotTo(HaveOccurred())
+	Expect(err).NotTo(HaveOccurred(), "failed to retrieve pod name for '%+v' : %s", pod, podNodeName)
```

Comment:
I feel this Expect() should be operated just after line 1345 for avoiding unnecessary podNodeName() call.